static/js/public/vueComponents/pagination.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+    <div class="columns pagination">
+        <div class="column is-12">
+            <ul v-if="numberOfRecords > 0">
+                <li v-if="currentPage !== 1"><a class="back-next" :href="pageUrl(currentPage - 1)"><<</a></li>
+                <li v-else class="no-link"><<</li>
+                <li v-for="page in totalPages">
+                     <a :href="pageUrl(page)" class="page-number" :class="{ current: currentPage === page }">{{ page }}</a>
+                </li>
+                <li v-if="currentPage < totalPages"><a class="back-next" :href="pageUrl(currentPage + 1)">>></a></li>
+                <li v-else class="no-link">>></li>
+            </ul>
+        </div>
+    </div>
+</template>
+
+<script>
+    define(['Vue'], function(Vue) {
+        Vue.component('pagination', {
+            template: template,
+
+            props: {
+                numberOfRecords: Number,
+                perPage: Number,
+                pageBaseUrl: String
+            },
+
+            data: function() {
+                return {
+                    currentPage: 1,
+                    totalPages: 1
+                }
+            },
+
+            methods: {
+                setPage: function() {
+                    let params = window.location.search;
+                    let page_number = params.split('=');
```

Comment:
Seems like this will run into problems when there are more than one GET parameter? Such as if the sort is included. Can you use `this.$route.query.page_number`?

Reply:
We're not currently using Vue router, so I don't think that will work. That said to get the testing the whole view setup will probably need to be overhauled.